daemon/package.json

```diff
@@ -42,6 +44,7 @@
     "rcon-srcds": "^2.1.0",
     "socket.io": "^4.6.1",
     "socket.io-client": "^4.7.5",
+    "utf-8-validate": "^6.0.4",
```

Comment:
Why install it? I didn't find you using it.

Reply:
Not installing this will result in errors with socket.io.